Include/internal/pycore_stackref.h

```diff
@@ -52,43 +53,91 @@ typedef union {
     uintptr_t bits;
 } _PyStackRef;
 
+struct _Py_stackref_entry {
+    PyObject *obj;
+    int is_live;
```

Comment:
This shouldn't be necessary, and AFAICT it isn't used.
The presence of a tag in the hashtable is equivalent to liveness.

Reply:
Good catch. I removed it.